### PR TITLE
Fixed repo_token being displayed in clear text

### PR DESCRIPTION
**Pull request by amercier**
Spaces after columns can be missing the JSON output. Because of that, `repo_token` value is not replaced by `[secure]`. See https://travis-ci.org/amercier/oeco/builds/47742447 as an example.

### CODE REVIEW

File: lib/coveralls/api.rb [hunk 1]
```diff
@@ -63,9 +63,9 @@ def self.hash_to_file(hash)
 
 		def self.apified_hash hash
 			config = Coveralls::Configuration.configuration
-			if ENV['CI'] || ENV['COVERALLS_DEBUG'] || Coveralls.testing
```
Comment (nickmerwin) [inline lib/coveralls/api.rb]: Hey why did you remove the ```ENV['CI']``` from this conditional?

Comment (amercier) [inline lib/coveralls/api.rb]: Ooops, my bad. That was for testing and I left it accidentally in the commit. Hold on, I'm going to remove it.

**Comment by amercier** [comment]
@nickmerwin Fixed :)

**Comment by amercier** [comment]
@nickmerwin What do you think about this? Do you think that can be merged?

**Comment by nickmerwin** [comment]
hi @amercier ah yes, thank you for pinging. I'll include in the next release (should be today).

**Comment by amercier** [comment]
:+1:
